core/src/main/java/com/datastax/oss/driver/internal/core/os/Native.java

```diff
@@ -35,6 +51,14 @@ public Libc load() {
         return new EmptyLibc();
       }
     }
+
+    private boolean isGraalBuildTime() {
```

Comment:
Is this https://github.com/datastax/java-driver/pull/1439/files#diff-357e4854869b2e21c38b1b437f11095aR130-R134 dependency always present if one of those settings evaluates to true?

Reply:
Yes, that's correct.  It has to be the case since at build-time the annotations are eval'd for C generation and at run-time the necessary infrastructure for the C calls has to be in place.